mpas_analysis/test/test_mpas_xarray.py

```diff
@@ -123,7 +123,17 @@ def test_selvals(self):
             ['time_avg_avgValueWithinOceanLayerRegion_avgLayerTemperature',
              'refBottomDepth']
 
-        selvals = {'nVertLevels': 0}
+        vertIndex = 10
```

Comment:
Why not just update old line 126 as `selvals = {'nVertLevels', 10}`?  Then we don't need new lines 126 and 136.  If you don't want to hard-code this make `nVertLevels=10` a default argument to `test_selvals` instead because this is cleaner.  Bottom line, we want to keep this testing code as terse as possible so that it is clear.